fix(coprocessor): update charts after rename

antoniupop opened the pull request.

antoniupop committed:
fix(coprocessor): update charts and documentation after rename

tawadaa reviewed (commented):
@antoniupop could you bump the chart version to `0.4.6` for the checks to pass?

tawadaa reviewed (commented):
For e2e tests, need to update host-listener and tfhe-worker docker image tags to point to the one with the renamed executables

antoniupop committed:
fix(coprocessor): fix dataflow graph finalization

antoniupop committed:
fix(coprocessor): fix uniqueness of the primary key on computations

Handles generated for trivial encrypt operations are not unique and
such ops require transaction ID as well to uniquely identify them.

antoniupop commented:
Passes e2e tests : https://github.com/zama-ai/fhevm/actions/runs/16747802730
